core/imageroot/var/lib/nethserver/node/bin/configure-home-basedir

```diff
@@ -0,0 +1,122 @@
+    else:
+        home_basedir = validate_home_basedir(args.DIR)
+        if not args.check_only:
+            store_configuration(home_basedir)
+            print("The base path for home directories has been updated.")
+
+def store_configuration(path):
+    if os.path.isdir("/sys/fs/selinux"):
+        # On systems with SELinux, configure path as /home equivalent
+        update_selinux_customization(path)
+    agent.set_env('HOME_BASEDIR', path)
+    print("HOME_BASEDIR=" + path)
+
+def update_selinux_customization(path):
+    ocurrent = subprocess.check_output(['semanage', 'fcontext', '-l', '-C'], text=True)
+    if not f"\n{path} = /home\n" in ocurrent:
+        subprocess.check_call(['semanage', 'fcontext', '-a', '-e', '/home', path])
+    subprocess.check_call(['restorecon', '-v', path])
+    update_parentdir_selinux_type(path)
+
+def update_parentdir_selinux_type(path):
+    """If needed, set home_root_t on parent dir as required by semanage-fcontext manpage."""
+    parentdir = os.path.dirname(path)
+    if parentdir != "/":
+        ols = subprocess.check_output(['ls', '-Zd', parentdir], text=True)
+        if ":default_t:" in ols:
+            subprocess.check_call(['semanage', 'fcontext', '-a', '-t', 'home_root_t', parentdir])
+            subprocess.check_call(['restorecon', '-v', parentdir])
+
+def validate_home_basedir(path):
+    """Validate the given path and return it in canonicalized form."""
+    if os.path.islink(path) or not os.path.isdir(path):
+        print(f"Error: {path} is not a directory.", file=sys.stderr)
+        sys.exit(2)
+    # Canonicalize the path value:
+    home_basedir = os.path.abspath(path)
+    if os.path.realpath(home_basedir) != home_basedir:
+        print(f"Error: {path} contains one or more symlink components.", file=sys.stderr)
+        sys.exit(2)
+    check_permissions(home_basedir)
+    check_unique_mountpoint(home_basedir)
+    return home_basedir
+
+def get_mountpoint_device(path):
+    """Check if path is a mountpoint and return its source device."""
+    try:
+        joutput = subprocess.check_output(['findmnt', '--json', '--mountpoint', path])
+        ofindmnt = json.loads(joutput)
+        return ofindmnt["filesystems"][0]["source"] # source device path
+    except subprocess.CalledProcessError:
+        print(f"Error: {path} is not a filesystem mountpoint.", file=sys.stderr)
+        sys.exit(2)
+
+def check_unique_mountpoint(path):
+    """Check if path is a mountpoint, and its source device is not
+    mounted elsewhere."""
+    device = get_mountpoint_device(path)
+    joutput = subprocess.check_output(['findmnt', '-o', 'TARGET', '--json', '--source', device])
+    ofindmnt = json.loads(joutput)
+    for ofs in ofindmnt["filesystems"]:
+        if ofs['target'] != path:
+            print(f"Error: device {device} has multiple mountpoints. It is mounted also on {ofs['target']}. Unmount it, persist the change, and retry.", file=sys.stderr)
+            sys.exit(2)
```

Comment:
Is this really needed? Someone may want to use a bind mount.
If there is no real trouble with Selinux, I'd remove this check.